src/test/java/org/sagebionetworks/web/unitclient/presenter/DownPresenterTest.java

```diff
@@ -108,20 +104,4 @@ public void testRepoUpHasLastPlace() {
 		verify(mockGlobalAppState).back();
 	}
 
-	@Test
```

Comment:
I think you should leave these two unit tests in (testRepoDown() and testCheckFailure())

Reply:
Thanks, I added it back.